web/src/App.tsx

```diff
@@ -23,8 +23,8 @@ import "antd/dist/antd.css";
 function AppRoutes() {
   const routes = useRoutes([
     { path: "/", element: <RootPage /> },
-    { path: "/dashboard", element: <DashboardPage /> },
-    { path: "/dashboard/:workspaceId", element: <DashboardPage /> },
+    { path: "/dashboard", element: <WorkspacePage /> },
+    { path: "/dashboard/:workspaceId", element: <WorkspacePage /> },
```

Comment:
Naming is getting a little jumbled. Page is Workspace, path is dashboard, selectedKey is Home. Please consolidate the naming

Reply:
@KaWaite , I renamed the key to `workspace`
The logic here is that the main dashboard route opens the workspace page.
The dashboard path is the main route, in the future, the main page may open other components.
I don't see that `/worksapce` would be a beautiful route for the main page, neither changing `Workspaepage` to `DashboardPage` is clear.